pom.xml

```diff
@@ -217,12 +200,6 @@
                   <goal>bower</goal>
                 </goals>
               </execution>
-              <execution>
-                <id>grunt build</id>
-                <goals>
-                  <goal>grunt</goal>
-                </goals>
-              </execution>
```

Comment:
The reason why I started running the Grunt build in this project is so that the tests could generate the minified file for js-cookie and run the tests against the minified file. At some point after these lines were added to java-cokie, [somebody created a PR](https://github.com/js-cookie/js-cookie/pull/221) to [only add source files to npm](https://github.com/js-cookie/js-cookie/blob/master/package.json#L27-L29) in the js-cookie project.

We're running the js-cookie tests in this project so that we don't duplicate the whole code. See https://github.com/js-cookie/js-cookie/blob/d2724a87a29cbf7ee539452fb42b3e3a1aff95a8/CONTRIBUTING.md#integration-with-server-side

Do you reckon it doesn't make sense to run the tests on the minified file? Why is the main reason for removing the `grunt build` execution goal?

Also, can you confirm that all the tests from js-cookie are still running after this PR?

Reply:
The Gruntfile here did not actually do anything, as far as I could see. We can specify the commands to run directly in the pom as well (to install the Javascript via bower), and this already happens. The tests are running fine, see Travis.

Reply:
Oh, I need to check the js-cookie tests as well, will do that.

Reply:
Cool, just confirm if the code was really to test the file in the minified form. If you're confident that it's not being used anywhere it should be ok to remove them

Reply:
Oh boy, this was a wild ride 😄 I fixed the integration test now as well and updated everything around it to recent components (Selenium, Wildfly) and it now works out-of-the-box when running `mvn verify`.

However, it does **not** test the minified form - which would need changes to `test/encoding.html` in the `js-cookie` repository, as only the non-minified version is included there. As nobody seems to have run these tests in ages, I think testing the non-minified form is more than OK for now!